documents/motions/09.2020.1 - Senior Delegates.md

```diff
@@ -4,21 +4,21 @@
 **Subject:** Senior Delegates  
 **Intent:** Procedure of acknowledgement of WCA Senior Delegates and description of their rights and duties  
 **Submitted by:** Board of Directors  
-**Date:** January 31, 2020  
+**Date:** September 30, 2020  
 
 # Motion
 
 Senior Delegates are WCA Staff and Full Delegates, with an additional role to manage the WCA Delegates in an attributed area.
 
 1. Senior Delegates have all the rights and duties of Full Delegates and extra rights and duties as described in this Motion.
 2. Rights of Senior Delegates
-   1. Senior Delegates can appoint Registered Speedcubers as Trainee Delegates or Junior Delegates in their attributed area.
-   2. Senior Delegates can promote Trainee Delegates to Junior Delegates in their attributed area.
-   3. Senior Delegates can promote Junior Delegates to Full Delegates in their attributed area.
-   4. Senior Delegates can appoint Delegates as Regional Delegates in their attributed area.
```

Comment:
I understand the change but I am not sure this is right, because it has the effect that board is influencing directly who has voting rights.

Reply:
This is just a clarification of the existing process. From Motion 08.2020.1:
6.3 Senior Delegates decide at any time which persons are proposed as WCA Delegates, then
send in an application for these persons to the WCA Board of Directors.
6.3.1 The WCA Board of Directors rejects or approves the application.
6.3.2 After approval, the Senior Delegate appoints the persons as Junior Delegates.

9.2.2 To promote a Junior Delegate to Full Delegate, the Senior Delegate must submit the
corresponding application to the WCA Board.
9.2.3 The WCA Board rejects or approves the application.
9.2.4 After approval, the Senior Delegate appoints the person as Full Delegate.

There is a similar description in other documents for promotion of Trainee and Regional Delegates, all requiring Board approval.

Reply:
> Isn't the Bylaws a separate document? I think there is a Bylaws document and there are Motions.
> https://www.worldcubeassociation.org/documents/WCA%20Bylaws.pdf
> 
> So it may be legally so that Bylaws are different from Motions.

The Bylaws themselves already describe how they should be amended:

> Except as otherwise provided by law, these Bylaws may be amended or repealed and new bylaws
adopted by approval of the Board, or by a supermajority (⅔) vote of the membership at a properly
announced meeting.

Currently, the Bylaws say they can be amended one way, but the Motions say the Bylaws are amended by a different process. We are keeping the amendment process from the Bylaws only so that the Bylaws and Motions do not contradict each other.